common/src/main/java/com/viaversion/viaversion/protocol/ProtocolPathEntryImpl.java

```diff
@@ -50,9 +51,7 @@ public boolean equals(final Object o) {
 
     @Override
     public int hashCode() {
-        int result = outputProtocolVersion;
-        result = 31 * result + protocol.hashCode();
-        return result;
+        return super.hashCode();
```

Comment:
This needs a proper hashcode impl, IJ can generate them, otherwise Objects.hash

Reply:
I generated it using IJ, but for some reason it didn't fill in the correct code lol, fixed